features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml

```diff
@@ -535,6 +535,14 @@
             Disable this config if additional params are not needed post logout redirect URL.
             -->
             <!--AllowAdditionalParamsFromPostLogoutRedirectURI>false</AllowAdditionalParamsFromPostLogoutRedirectURI-->
+
+            <!--
+                FAPI Profile Support.
+            -->
+            <!-- <FAPI>
+                <EnableCibaProfile>{{fapi.enable_ciba_profile}}</EnableCibaProfile>
+                <EnableSecurityProfile>{{fapi.enable_security_profile}}</EnableSecurityProfile>
```

Comment:
Why we need placeholders in the xml file

Reply:
we should have default values here.

Reply:
Fixed